### PR TITLE
bpkg: add page

### PR DESCRIPTION
**Pull request by vitorhcl**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):** 1.1.3

### CODE REVIEW

File: pages/common/bpkg.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+`bpkg update`
+
+- Install a package [g]lobally:
+
+`bpkg install --global {{package_name}}`
+
+- Install a package in a subdirectory of the current directory:
+
+`bpkg install {{package_name}}`
```
Comment (kbdharun) [inline pages/common/bpkg.md:16]:
```suggestion
`bpkg install --global {{package}}`

- Install a package in a subdirectory of the current directory:

`bpkg install {{package}}`
```

File: pages/common/bpkg.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+- Install a specific version of a package:
+
+`bpkg install {{package_name}}@{{version}} -g`
+
+- Show details about a specific package:
+
+`bpkg show {{package_name}}`
```
Comment (kbdharun) [inline pages/common/bpkg.md:24]:
```suggestion
`bpkg install {{package}}@{{version}} -g`

- Show details about a specific package:

`bpkg show {{package}}`
```

File: pages/common/bpkg.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+- Run a command, optionally specifying its arguments:
+
+`bpkg run {{command}} {{command_arg1 command_arg2 ...}}`
```
Comment (kbdharun) [inline pages/common/bpkg.md:28]:
```suggestion
`bpkg run {{command}} {{argument1 argument2 ...}}`
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

We generally use `package` instead of `package_name` in placeholders.

File: pages/common/bpkg.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
+# bpkg
+
+> A package manager for Bash scripts.
+> More information: <https://github.com/bpkg/bpkg>.
+
+- Update the local index:
+
+`bpkg update`
+
+- Install a package [g]lobally:
```
Comment (fazlearefin) [inline pages/common/bpkg.md:10]:
```suggestion
- Install a package globally:
```

Short options only require `[]` when possible. `--global` is not a short option.

**Review by kbdharun — APPROVED** [review]

File: pages/common/bpkg.md (hunk 4 above)
Comment (kbdharun) [reply pages/common/bpkg.md:10]: > Short options only require [] when possible. --global is not a short option.

We allow mnemonics to be associated with words explaining to context too.

Comment (fazlearefin) [reply pages/common/bpkg.md]: This was discussed in a previous PR. Let me find it and update this comment with the link.

Comment (kbdharun) [reply pages/common/bpkg.md]: > This was discussed in a previous PR. Let me find it and update this comment with the link.

Yeah, I recollected the discussion about it; and applied your suggestion.

**Review by fazlearefin — APPROVED** [review]

**Pull request merged**